packages/edit-post/src/store/reducer.js

```diff
@@ -111,19 +102,6 @@ export const preferences = flow( [
 
 		return state;
 	},
-	pinnedPluginItems( state, action ) {
```

Comment:
This is going to remove the list of pinned/unpinned items stored in the browser for users.

Reply:
Yes, the current preferences are removed with this change. I think it is not a big issue.